integrations/tensorflow/.bazelrc

```diff
@@ -18,3 +18,36 @@ try-import %workspace%/../../user.bazelrc
 
 # Include settings from the integrations/tensorflow directory too.
 try-import %workspace%/tf-user.bazelrc
+
+build --action_env PYTHON_BIN_PATH="/usr/bin/python"
+build --action_env PYTHON_LIB_PATH="/usr/lib/python3.9/dist-packages"
+build --python_path="/usr/bin/python"
+
+# Generated interactively using Tensorflow ./configure script.
+# https://www.tensorflow.org/lite/guide/build_android#configure_workspace_and_bazelrc
+build --action_env ANDROID_NDK_HOME="/usr/local/google/home/mariewhite/Android/Sdk/ndk/20.1.5948944"
+build --action_env ANDROID_NDK_API_LEVEL="21"
+build --action_env ANDROID_BUILD_TOOLS_VERSION="30.0.0"
+build --action_env ANDROID_SDK_API_LEVEL="29"
+build --action_env ANDROID_SDK_HOME="/usr/local/google/home/mariewhite/Android/Sdk"
```

Comment:
These are specific for you platform. Did you intend to have this leaked through? We might be able to fix up the .bazelrc generation so this can be configured per user.